app/models/comment.rb

```diff
@@ -0,0 +1,13 @@
+class Comment < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+
+  after_save :update_post_comments_count
+
+  private
+
+  def update_post_comments_count
+    post_comments_count = Comment.where(post_id:).count
+    post.update(commentsCounter: post_comments_count)
```

Comment:
:red_circle: [REQUIRED] Please ensure that you adhere to Ruby's best practices for variable naming. The naming convention for variables is` snake_case` for methods and variables. So `commentsCounter` should be `comments_counter`